app/models/view/alerts/Alert.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package models.view.alerts;
+
+import com.arpnetworking.logback.annotations.Loggable;
+import com.google.common.collect.ImmutableMap;
+import models.internal.alerts.AlertEvaluationResult;
+import models.internal.scheduling.JobExecution;
+
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * View model for an alert.
+ * <p>
+ * Play view models are mutable.
+ *
+ * @author Christian Briones (cbriones at dropbox dot com)
+ */
+@Loggable
+public final class Alert {
+    private UUID _id;
+    private String _name;
+    private String _description;
+    private boolean _enabled;
+    private ImmutableMap<String, Object> _additionalMetadata;
```

Comment:
Hrm. This contradicts the statement above: `Play view models are mutable.` (the original did too)

Reply:
I can change this but I didn't see it as a contradiction - it's not _deeply_ mutable, but you can still swap out the map reference here with something else.

Reply:
Not sure if needs changing. Like I said it's everywhere. Maybe add a comment. I don't think it's a big deal at this point.